Dockerfile

```diff
@@ -11,20 +11,20 @@ RUN rm -rf /var/cache/apk/*
 WORKDIR /app
 
 # NPM
-COPY --chown=node:node package.json .
-COPY --chown=node:node package-lock.json .
-RUN npm ci --legacy-peer-deps
+COPY --chown=node:node package*.json .
+RUN npm ci --legacy-peer-deps --ignore-scripts
 RUN npm cache clean --force
 
-COPY --chown=node:node . .
+COPY --chown=node:node src src
+COPY --chown=node:node tsconfig*.json .
 
 # Build with all dependencies
 RUN npm run build
 
 # Lean NPM - Only install `dependencies`
 # `devDependencies` will still be resolved inside `package-lock.json`,
 # however they will not be installed inside `node_modules` directory.
-RUN npm ci --legacy-peer-deps --omit=dev
+RUN npm ci --legacy-peer-deps --omit=dev --ignore-scripts
```

Comment:
I have tried this in the past and it has caused `npm run build` to fail, were you able to build the container without any errors locally?

Reply:
I've tried it locally and it's working for me. I wonder if it's WSL otherwise?

Reply:
never mind. it crashed now...
I've removed it